erna/datacheck_conditions.py

```diff
@@ -1,7 +1,20 @@
 conditions = dict()
 
+def create_condition_set(conditionset=['standard']):
```

Comment:
shouldn't `standard` start with `@`?

Reply:
Yes, fixed it also at all the other places.